src/components/connection-prompt/bluetooth/ConnectBatteryDialog.svelte

```diff
@@ -0,0 +1,31 @@
+<!--
+  (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+ 
+  SPDX-License-Identifier: MIT
+ -->
+
+<script lang="ts">
+  import StandardButton from '../../StandardButton.svelte';
+  import { t } from '../../../i18n';
+
+  export let onNextClick: () => void;
+  export let onBackClick: () => void;
+</script>
+
+<main>
+  <div class="w-180 leading-10 pb-5">
+    <p class="font-bold text-2xl text-left pb-3">
+      {$t('connectMB.connectBattery.heading')}
+    </p>
+    <p>{$t('connectMB.connectBattery.subtitle')}</p>
+    <img
+      src="public/imgs/stylised-microbit-connected.svg"
```

Comment:
Path for images should start at /imgs
```suggestion
      src="/imgs/stylised-microbit-connected.svg"
```